### ISSUE TITLE
NPE

### ISSUE THREAD

**Issue by x1p** [Esmorall/EntityTrackerFixer#6]
FYI: Saw this little error just after someone joined.
Probably innocent, but just to inform you.
```
[10:46:58 WARN]: [EntityTrackerFixer] Task #22 for EntityTrackerFixer v1.2.1 generated an exception
java.lang.NullPointerException: null
at net.minemora.entitytrackerfixer.v1_14_R1.entityTick.EntityTickManager.disableTicking(EntityTickManager.java:45) ~[?:?]
at net.minemora.entitytrackerfixer.v1_14_R1.tasks.UntrackerTask.untrackProcess(UntrackerTask.java:96) ~[?:?]
at net.minemora.entitytrackerfixer.v1_14_R1.tasks.UntrackerTask.run(UntrackerTask.java:45) ~[?:?]
at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[purpur.jar:git-Purpur-336]
at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:452) ~[purpur.jar:git-Purpur-336]
at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1178) ~[purpur.jar:git-Purpur-336]
at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:426) ~[purpur.jar:git-Purpur-336]
at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1105) ~[purpur.jar:git-Purpur-336]
at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:932) ~[purpur.jar:git-Purpur-336]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```
Btw: purpur is a fork of paper.

**Comment by Elecast2** [comment]
Hi, are you using new experimental option? i will post an update with some bug fixes soon

**Comment by rayanbzd** [comment]
Hello,

The bug makes the plugin crash and no longer works after error and need to reload it... It happens when you have additional worlds that have been loaded from plugins such as Multiverse-Core because the world loading appear after cache load of EntityTrackerFixer. 

This bug is patched on the PullRequest #11 if u want to test it, there is a jar.

**Comment by Elecast2** [comment]
EntityTrackerFixer no longer loads cache of worlds on-load so this error shouldn't happen anymore.
